CMakeLists.txt

```diff
@@ -602,7 +643,7 @@ if(PLAYER_BUILD_LIBLCF)
 	target_link_libraries(${PROJECT_NAME} lcf)
 else()
 	# Use system package
-	find_package(liblcf REQUIRED)
+	find_package(liblcf REQUIRED CONFIG)
```

Comment:
This is actually an unrelated change: We always want a recent liblcf and we always ship a config file, so we can skip all `Findliblcf` stuff.